app/config/runtimes/specifications.php

```diff
@@ -6,7 +6,7 @@
     Specification::S_05VCPU_512MB => [
         'slug' => Specification::S_05VCPU_512MB,
         'memory' => 512,
-        'cpus' => 1 // TODO: revert this, it's a temporary change to test function performance.
+        'cpus' => 0.5
```

Comment:
I think it was increased due to waterfall effect on Cloud.
Let's make sure with Christy that everyone was migrated to `S_1VCPU_512MB` before releasing this.